netbox/tenancy/models/contacts.py

```diff
@@ -84,24 +84,28 @@ class Contact(PrimaryModel):
     )
 
     clone_fields = (
-        'group', 'name', 'title', 'phone', 'email', 'address', 'link',
+        'groups', 'name', 'title', 'phone', 'email', 'address', 'link',
     )
 
     class Meta:
         ordering = ['name']
-        constraints = (
-            models.UniqueConstraint(
-                fields=('group', 'name'),
-                name='%(app_label)s_%(class)s_unique_group_name'
-            ),
-        )
         verbose_name = _('contact')
         verbose_name_plural = _('contacts')
 
     def __str__(self):
         return self.name
 
 
+class ContactGroupMembership(models.Model):
```

Comment:
We probably want to add this to `__all__` so it can be imported from `tenancy.models`.